tests/server_test_cases/config_tests.py

```diff
@@ -440,7 +440,7 @@ def test_analytics_id(self):
         config.set(analytics_id='analytics_id_xyz')
 
         doc = self.go('/haiti/create')
-        assert "getTracker('analytics_id_xyz')" in doc.content
+        assert "gaProperty': 'analytics_id_xyz'" in doc.content
```

Comment:
That's weird, maybe the test resource_tests.py is flaky. Can you run it multiple times to see if it passes? The code itself looks good, so you can merge if all_test passes.